swe_cp_WB.m

```diff
@@ -619,6 +640,21 @@ function swe_cp_WB(SwE)
 
   end
   
+  %-Initialise parametric TFCE results images, if TFCE has been selected.
+  %---------------------------------------------------------------------- 
+  if TFCE
+      VlP_tfce_pos = swe_create_vol(sprintf('swe_tfce_lp-WB_c%02d%s', 1, file_ext), DIM, M,...
+                  'Original parametric TFCE statistic data.'); 
+      VlP_tfce_FWE_pos = swe_create_vol(sprintf('swe_tfce_lpFWE-WB_c%02d%s', 1, file_ext), DIM, M,...
+                  'Original parametric TFCE statistic data.'); 
```

Comment:
The description isn't correct... this is "FWE-corrected nonparametric TFCE p-value" or whatever matches a similar description for other similar files, right?